src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -26,7 +27,7 @@ internal class HttpApiHandler
         private const string ShowHistoryOutputParameter = "showHistoryOutput";
         private const string CreatedTimeFromParameter = "createdTimeFrom";
         private const string CreatedTimeToParameter = "createdTimeTo";
-        private const string RuntimeStatusParameter = "runtimeStatus";
+        private const string RuntimeStatusParameter = "runtimeStatus[]";

```

Comment:
I don't think we should change this query string parameter name. I have never seen this convention.